umbraco/Umbraco-CMS
# rename block grid allowance to permissions

### Prerequisites


Issue reported on Discord in #hacktoberfest channel.
When configuring the Block Grid Editor, there is a panel called "Allowance",  it would be more consistent to use the word "Permissions" like everywhere else in the doc type editor.

### Description

**What did you do?**
Renamed the panel in the block grid editor for Allowance to Permissions in the default view, and in the en and en_us localization labels.

**Why did you do it?**
Hacktoberfest! :) 

Screenshot of fix:
![blockgrid-panel-name](https://user-images.githubusercontent.com/4968254/198814718-87e7af51-110c-45de-abf4-1e70540271ef.png)

How to test the changes:
1. Install v11
2. Create doctype and add block grid property editor with demo configurations
3. Click on demo block
4. Verify the "configuration" panel no longer says Allowance and instead reflects Permissions
